website/docs/cloud-docs/integrations/kubernetes/annotations-and-labels.mdx

```diff
@@ -0,0 +1,19 @@
+---
+page_title: Annotations and Labels used by Terraform Cloud Operator
+description: >-
+  Full list of annotations and labels used by Terraform Cloud Operator.
+---
+
+# Annotations and Labels used by Terraform Cloud Operator
+
+## Annotations
+
+| Annotation key | Target resources | Possible values | Description |
+| --- | --- | --- | --- |
+| `workspace.app.terraform.io/run-new` | Workspace | "true" | Set this annotation to `"true"` to trigger a new run. Example: `kubectl annotate workspace <WORKSPACE-NAME> workspace.app.terraform.io/run-new="true"`. |
+| `workspace.app.terraform.io/run-type` | Workspace | `plan`, `apply`, `refresh` | Specify the run type. More information: [Run Modes and Options](https://developer.hashicorp.com/terraform/cloud-docs/run/modes-and-options). It defaults to `plan`. Changing this annotation **DOES NOT** trigger a new run. |
+| `workspace.app.terraform.io/run-terraform-version` | Workspace | valid Terraform version | Specify the Terraform version to use. This setting works only when the annotation `workspace.app.terraform.io/run-type` is set to `plan`. It defaults to the Workspace version. Changing this annotation **DOES NOT** trigger a new run. |
```

Comment:
```suggestion
| `workspace.app.terraform.io/run-terraform-version` | Workspace | Any valid Terraform version | Specifies the Terraform version to use. Changing this annotation does not start a new run. Only valid when the annotation `workspace.app.terraform.io/run-type` is set to `plan`. Defaults to the Workspace version. |
```

Reply:
Is only a specific version valid (ie. "1.0.0") or can the user define a version constraint (ie. "~> 1.0.0")? If the latter, I think we should add the following sentence after "Specifies the Terraform version to use.":

"This can be either an exact version or a [version constraint](https://developer.hashicorp.com/terraform/language/expressions/version-constraints) (like ~> 1.0.0)."